### DISCUSSION TITLE
New custom select menu

### DISCUSSION THREAD

**Discussion by devopg** [saadeghi/daisyui#2249] [Ideas / feature requests]
Hello. I am gonna assume its time to integrate a new cool component for custom select menu - new tag: <selectmenu>
You can see more here about it 
https://developer.chrome.com/blog/whats-new-css-ui-2023/#selectmenu

![image](https://github.com/saadeghi/daisyui/assets/61819835/1787bd2f-1b48-46c5-ab8b-9b3adc681364)

**Comment by saadeghi** [comment]
How's the browser support?
